browser/html/cool.html.m4

```diff
@@ -283,6 +283,7 @@ m4_ifelse(MOBILEAPP,[true],
             <div style="margin-inline-end: auto;"><div id="lokit-version" dir="ltr"></div></div>
             m4_ifelse(MOBILEAPP,[],[<div id="served-by"><span id="served-by-label"></span>&nbsp;<span id="os-info"></span>&nbsp;<wbr><span id="coolwsd-id"></span></div>],[<p></p>])
             <div id="slow-proxy"></div>
+            <div id="js-dialog">JSDialogs: <a href="javascript:void(app.socket.sendMessage('uno .uno:WidgetTestDialog'))">View widgets</a></div>
```

Comment:
I think we want it only in DEBUG build, could you check if there is some macro for it?
Like above for mobile app: `m4_ifelse(MOBILEAPP`